src/storage/src/healthcheck.rs

```diff
@@ -41,9 +41,12 @@ pub async fn write_to_persist(
     let mut handle = client
         .open_writer(
             status_shard,
-            &format!("healthcheck::write_to_persist {}", collection_id),
             Arc::new(relation_desc.clone()),
             Arc::new(UnitSchema),
+            Diagnostics {
+                shard_name: status_shard.to_string(),
```

Comment:
am I reading this correctly that this sets the name to the ShardId? that doesn't seem quite right

Reply:
It'll need to be the `status_collection_id` I believe, which may currently only be in the controller, not storage :(

Reply:
yes you're both right 🤦 

plumbing this through is nontrivial, going to leave a TODO and name it `status($sink_id)` for now